src/Options.php

```diff
@@ -1154,6 +1190,14 @@ public function validateRemoteUri(string $uri)
             return [false, "Remote file requested, but remote file download is disabled."];
         }
 
+        if (is_array($this->allowedRemoteHosts)) {
+            $host = parse_url($uri, PHP_URL_HOST);
+
+            if (!in_array($host, $this->allowedRemoteHosts, true)) {
```

Comment:
I wonder if we should `mb_strtolower` these values before comparison since case doesn't matter for host resolution.